CHANGELOG.md

```diff
@@ -27,8 +27,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - R manifest package generation now uses only `renv.lock` file by default. To use the
-  local `renv` library instead, set `[r].packages_from_library = true` in the
+  local `renv` library instead, set `r.packages_from_library = true` in the
```

Comment:
Unrelated, but it was requested in a previous PR to change this and it didn't value a dedicated PR.